DataGrid/TreeList: add the Column Chooser and Header Filter new features (#4896)
DevExpress/devextreme-documentation: api-reference/40 Common Types/15 grids/ColumnChooserSelectionConfig/allowSelectAll.md

---
---
##### shortDescription

> Allows users to simultaneously select all items.

Reviewer: It's true if this property is set to true. I would rephrase it to something like "Specifies whether to allow..."

> Specifies whether to allow users to simultaneously select all items.

Reviewer 2: It's not clear what you mean by "simultaneously" - the act of selection or the selection state:

- I can select all items with a single operation (let's say I press CTRL+A and select all ... simultaneously)
- I can select items one by one, and the option tells me if can ultimately select all of them. 

I guess it's the first option, because the second doesn't make much sense, but then I'd write something like this: 

 ```suggestion
Specifies whether a "Select All" option is available to users.
```
